src/EventHub/EventHub.Autorest/docs/Get-AzEventHubPrivateEndpointConnection.md

```diff
@@ -0,0 +1,195 @@
+---
+external help file:
+Module Name: Az.EventHub
+online version: https://learn.microsoft.com/powershell/module/az.eventhub/get-azeventhubprivateendpointconnection
+schema: 2.0.0
+---
+
+# Get-AzEventHubPrivateEndpointConnection
```

Comment:
There is already a doc for the cmdlet Get-AzEventHubPrivateEndpointConnection https://github.com/Azure/azure-powershell/blob/generation/src/EventHub/EventHub.Autorest/docs/Get-AzEventHubPrivateEndpointConnection.md. Do you know why this might be showing as a newly added file ?